projects/lights_control/src/lights_gpio.c

```diff
@@ -0,0 +1,212 @@
+#include "lights_gpio.h"
+#include "event_queue.h"
+#include "gpio.h"
+#include "gpio_mcu.h"
+#include "lights_events.h"
+#include "status.h"
+
+#define LIGHTS_HORN_PERIPHERAL 0
+#define NUM_LIGHTS_HORN_PERIPHERALS 1
+#define LIGHTS_STROBE_PERIPHERAL 0
+#define NUM_LIGHTS_STROBE_PERIPHERALS 1
+
+typedef enum {
+  LIGHTS_HIGH_BEAMS_RIGHT_PERIPHERAL = 0,
+  LIGHTS_HIGH_BEAMS_LEFT_PERIPHERAL,
+  NUM_LIGHTS_HIGH_BEAMS_PERIPHERALS
+} LightsHighBeamsPeripherals;
```

Comment:
What's the benefit of having so many enums within this source file? It isn't exposed anywhere.

Reply:
yup changed it 😛